src/pwa-install.ts

```diff
@@ -39,7 +39,7 @@ export class pwainstall extends LitElement {
     return css`
       :host {
         --install-focus-color: #919c9c;
-        --install-button-color: #1fc2c8;
```

Comment:
this color is defined below for the background of the ids already listed, if another color is preferable let me know

Reply:
I think were good with this one, thanks @lee-leonardo !